src/Repository.php

```diff
@@ -553,6 +553,18 @@ public function notActive($name)
     }
 
     /**
+     * Determine if the given module exist and is activated.
+     *
+     * @param string $name
+     *
+     * @return bool
+     */
+    public function isActive($name)
+    {
+        return ($module = $this->find($name)) && $module->active();
```

Comment:
I understand your point, but the main purpose of laravel is easy coding.
So if you need a fast validation, this:
```php
if (Module::isActive('module') {
    // do something
}
```
is faster that:
```php
try {
    if (Module::active('module') {
        // do something
    }
} catch (ModuleNotFoundException $e) {
}
```
that for every module I use.

That behavior make me add this extra code in every module for exceptions that I don't need to catch, I just want to ignore some code if the module don't exist or is disabled.